scm/driver/harness/webhook.go

```diff
@@ -356,15 +358,27 @@ func convertPullReq(pr pullReq, ref ref, commit hookCommit) scm.PullRequest {
 	return scm.PullRequest{
 		Number: pr.Number,
 		Title:  pr.Title,
+		Body:   pr.Description,
 		Closed: pr.State != "open",
 		Source: pr.SourceBranch,
 		Target: pr.TargetBranch,
 		Merged: pr.State == "merged",
 		Fork:   "fork",
```

Comment:
shouldn't this be empty 🤔

Reply:
https://github.com/drone/go-scm/blob/ecd9254d1d864e74c348bfa8625c63701aa0f691/scm/driver/github/pr.go#L78

Reply:
not sure I understand - that's pointing to "body" field.
also checked the code there, it's using reponame of head:
<img width="398" alt="image" src="https://github.com/user-attachments/assets/c05b8ef5-cb76-4030-86f7-2ac7b22e13ca" />

Reply:
![Screenshot 2025-04-28 at 1 29 15 PM](https://github.com/user-attachments/assets/f6bccc71-a4aa-43c4-9e07-0cb1b66cc55a)

Reply:
https://www.perplexity.ai/search/what-does-body-and-sha-in-base-UeAsZMv2Q5S9qrkUnKQOjg

Reply:
my comment was on `fork`, not body

Reply:
ok that wasnt touched by me in this PR. will have to make changes later if that is ok ?